Deploy/stacks/dynamic/stack-clients/pom.xml

```diff
@@ -106,6 +106,12 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
note that if you change this, you'll have to update the version numbers, including the stack-manager, data uploader that are dependent on this

Reply:
Those changes are required to build the Stack Client properly to be packaged for py4jps wrapping. Can you please point me to what exactly needs to be changed or commit those changes directly on this branch as part of this PR?

Reply:
I'll make the changes and commit them.